.github/workflows/test-application.yaml

```diff
@@ -189,7 +189,6 @@ jobs:
                       php-extensions: 'ctype, iconv, mysql, imagick'
                       tools: 'composer:v2'
                       composer-stability: 'dev'
-                      composer-options: '--ignore-platform-reqs'
```

Comment:
this is only required for SUlu 2.5 and so can be removed on 2.6: added by https://github.com/sulu/sulu/pull/7644